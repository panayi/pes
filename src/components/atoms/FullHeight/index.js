/* @flow */
import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

type Props = {
  children: React$Node,
};

const FullHeight = ({ children }: Props) => (
  <div className="full-height">
    {children}
  </div>
);

FullHeight.propTypes = {
  children: PropTypes.node,
};

FullHeight.defaultProps = {
  children: null,
};

export default FullHeight;
