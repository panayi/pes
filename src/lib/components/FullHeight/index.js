import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

const FullHeight = ({ children }) => (
  <div className="full-height">
    {children}
  </div>
);

FullHeight.propTypes = {
  children: PropTypes.node,
};

FullHeight.defaultProps = {
  children: null,
};

export default FullHeight;
