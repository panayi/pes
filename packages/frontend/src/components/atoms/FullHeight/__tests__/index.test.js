import React from 'react';
import FullHeight from '../index';

describe('[Component] FullHeight', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <FullHeight>
        <header>I am a header</header>
        <article>I am an article</article>
      </FullHeight>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
