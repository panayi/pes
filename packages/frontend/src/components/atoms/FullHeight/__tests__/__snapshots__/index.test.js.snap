// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] FullHeight should render correctly 1`] = `
<div
  className="full-height"
>
  <header>
    I am a header
  </header>
  <article>
    I am an article
  </article>
</div>
`;
